src/core/RenderTarget.js

```diff
@@ -204,6 +202,37 @@ class RenderTarget extends EventDispatcher {
 
 	}
 
+	_setTextureOptions( options = {} ) {
+
+		const values = {
+			minFilter: LinearFilter,
+			generateMipmaps: false,
+			flipY: false,
+			internalFormat: null
+		};
+
+		if ( options.mapping !== undefined ) values.mapping = options.mapping;
+		if ( options.wrapS !== undefined ) values.wrapS = options.wrapS;
+		if ( options.wrapT !== undefined ) values.wrapT = options.wrapT;
```

Comment:
For all render target types relying on 3D or array textures, we must honor the `wrapR`.